### PR TITLE
[frontend] Enable custom name for Hive interpreters

### PR DESCRIPTION
**Pull request by JohanAhlen**
## What changes were proposed in this pull request?

- (Please fill in changes proposed in this fix)

## How was this patch tested?

- (Please explain how this patch was tested. Ex: unit tests, manual tests)
- (If this patch involves UI changes, please attach a screen-shot; otherwise, remove this)

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

File: desktop/core/src/desktop/conf.py [hunk 1]
```diff
@@ -1788,6 +1788,13 @@ def get_instrumentation_default():
   help=_('Choose whether to pick configs only from [desktop] [[cluster]]')
 )
 
+ENABLE_UNIFIED_ANALYTICS = Config(
```
Comment (Harshg999) [inline desktop/core/src/desktop/conf.py:1791]: Also add the config flag in `pseudo-distributed.ini.tmpl` and `hue.ini`?

Comment (romainr) [reply desktop/core/src/desktop/conf.py:1791]: When too CLDR specific, often we skip it

**Review by romainr — APPROVED** [review]
Nice!

Maybe could have used UNIFIED_ANALYTICS_NAME but we actually can always grep for it, and using a flag allow other tweaks/magic than name down the road.

File: desktop/libs/notebook/src/notebook/conf.py [hunk 2]
```diff
@@ -128,6 +128,7 @@ def get_ordered_interpreters(user=None):
 
   return [{
       "name": i.get('nice_name', i['name']),
+      'displayName': 'Unified Analytics' if ENABLE_UNIFIED_ANALYTICS.get() and i.get('dialect', i['name']).lower() == 'hive' else i.get('nice_name', i['name']),
```
Comment (romainr) [inline desktop/libs/notebook/src/notebook/conf.py:131]: To inject there IIRC:
```
Traceback (most recent call last):
  File "/home/runner/work/hue/hue/build/env/lib/python3.8/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/runner/work/hue/hue/desktop/libs/notebook/src/notebook/api_tests.py", line 691, in test_get_interpreters_to_show
    assert_equal(
AssertionError: Lists differ: [{'name': 'Hive', 'interface': 'hiveserver2', 'type': 'hive'[641 chars]va'}] != [{'name': 'Hive', 'displayName': 'Hive', 'type': 'hive', 'in[733 chars]lse}]

First differing element 0:
{'name': 'Hive', 'interface': 'hiveserver2', 'type': 'hive'[115 chars]ive'}
{'name': 'Hive', 'displayName': 'Hive', 'type': 'hive', 'in[138 chars]alse}

```

File: desktop/core/src/desktop/conf.py (hunk 1 above)
Comment (Harshg999) [reply desktop/core/src/desktop/conf.py:1791]: Ohh I see!

**Comment by JohanAhlen** [comment]
> Nice!
> 
> Maybe could have used UNIFIED_ANALYTICS_NAME but we actually can always grep for it, and using a flag allow other tweaks/magic than name down the road.

Yeah, I thought about and tried some options, and most clean to go with a temporary flag. Optimally I think a custom connector would be the way to go and then drop the flag (now easy to grep for removal).

**Pull request merged**